app/components/works/edit/citation_component.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= tag.div data: { controller: 'citation' } do %>
+  <p><%= t('works.edit.fields.citation.help_text') %></p>
+
+  <%= tag.div class: 'form-check mb-4' do %>
+    <%= form.radio_button :auto_generate_citation, true, class: 'form-check-input', checked: true, data: { citation_target: 'autoGenerateCitation', action: 'click->citation#autoGenerateCitationSelected' } %>
```

Comment:
For the data-actions here, I might prefer name that is more descriptive of what they do (vs. some reflection of form state). E.g., `disableCustomCitation` & `enableCustomCitation` or some such?